src/OpenTracing/Tracer.php

```diff
@@ -54,9 +95,8 @@ public function extract($format, Reader $carrier);
      *
      * This method might not be needed depending on the tracing implementation
      * but one should make sure this method is called after the request is finished.
-     * As an implementor, a good idea would be to use an asynchronous message bus
-     * or use the call to fastcgi_finish_request in order to not to delay the end
-     * of the request to the client.
+     * As an implementor, a good idea would be to use register_shutdown_function
+     * or fastcgi_finish_request in order to not to delay the end of the request to the client.
```

Comment:
This is not an "or" imho, since you can use both. register_shutdown_function is still waiting to finish the communication with the Webserver if you don't use fastcgi_finish_request.